src/templates/powered_by_template.html

```diff
@@ -0,0 +1,40 @@
+<html>
+    <head>
+        <title>{{title}}</title>
+        <link rel="stylesheet" type="text/css" href="static/styles.css">
+    </head>
+    <body>
+        <h1>{{title}}</h1>
+        <p>{{num_configs}} collections.</p>
```

Comment:
Should probably be `actions`?

Reply:
Yeah, that aligns better with the rest of the naming